README.md

```diff
@@ -1,4 +1,10 @@
-[![Maintained by Gruntwork.io](https://img.shields.io/badge/maintained%20by-gruntwork.io-%235849a6.svg)](https://gruntwork.io/?ref=repo_aws_vault)
+# DISCLAIMER
+This repository is no longer supported, please consider using [this repository](https://registry.terraform.io/modules/hashicorp/vault-starter/aws/latest) for the latest and most supported version for Vault.
```

Comment:
Oh, one more NIT: put the first sentence in bold. I think you did that already in some of the other PRs.